{{cookiecutter.plugin_name}}/.github/workflows/test_and_deploy.yml

```diff
@@ -84,13 +84,13 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -U setuptools setuptools_scm wheel twine
+          pip install -U setuptools setuptools_scm wheel twine build
       - name: Build and publish
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.TWINE_API_KEY }}
         run: |
           git tag
-          python setup.py sdist bdist_wheel
+          python -m build .
```

Comment:
Does this build a wheel by default? Or where is this decision/configuration now made?

Reply:
yep, `python -m build .` is equivalent to `python setup.py sdist bdist_wheel` 

from https://pypa-build.readthedocs.io/en/latest/#python--m-build

> By default, a source distribution (sdist) is built from {srcdir} and a binary distribution (wheel) is built from the sdist. This is recommended as it will ensure the sdist can be used to build wheels. Pass -s/–sdist and/or -w/–wheel to build a specific distribution.